contracts/src/CrossChainApplications/NativeTokenBridge/README.md

```diff
@@ -0,0 +1,34 @@
+# Native-to-Native Token Bridge
+
+A pair of smart contracts built on top of Teleporter to support using an ERC20 token or the native token of any `subnet-evm` chain as the native token for a given subnet.
+
+## Design
+The native-to-native bridge is implemented using two primary contracts.
+- `NativeTokenSource`
+    - Lives on the `Source chain`. Pairs with exactly one `NativeTokenDestination` contract on a different chain.
```

Comment:
```suggestion
    - Lives on the Source chain. Pairs with exactly one `NativeTokenDestination` contract on a different chain.
```

This is not code-formatted elsewhere. Let's be consistent with how we refer to the Source and Destination chains in this README, including code-formatting and capitalization.